src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Completion.cs

```diff
@@ -45,6 +53,19 @@ public enum CompletionItemKind
             Reference = 18
     }
 
+    public class Command
+    {
+        /// <summary>
+        /// The identifier of the actual command handler, like `vsintellicode.completionItemSelected`.
+        /// </summary>
+        public string command;
```

Comment:
public variables should Uppercase, e.g. Command.  Ideally we'd use properties instead of fields if there is not reason not too.

Reply:
Changed to properties.